.github/workflows/master-mirrors-main.yaml

```diff
@@ -0,0 +1,19 @@
+name: 'sync master with main'
+
+on:
+  push:
+    branches:
+      - 'main'
+
+jobs:
+  push-master:
+    runs-on: ubuntu-latest
+    steps:
+    - name: Checkout repository
+      uses: actions/checkout@v2
+      with:
+        fetch-depth: 0
+    - name: Push HEAD to master
+      # Deliberately not force-pushing: designed to fail if this would mean rewriting history
```

Comment:
question, non-blocking: how difficult would it be to add a Slack or email notification for @Kong/team-k8s if this ever failed?

Reply:
I believe we get this by default more or less, though it doesn't go to the entire team. GitHub action failures natively send an email alert to whomever took the action that initiated when they fail, e.g. I receive an email if the existing action to release charts on main push fails after I merge a PR to main. That might be a per-account preference, not sure.